embassy-stm32/src/i2c/v2.rs

```diff
@@ -111,6 +99,7 @@ impl<'d, T: Instance> I2c<'d, T> {
                 w.set_nbytes(length as u8);
                 w.set_start(i2c::vals::Start::START);
                 w.set_autoend(i2c::vals::Autoend::AUTOMATIC);
+                w.set_autoend(stop.autoend());
```

Comment:
The previous set_autoend does nothing then, it should be removed.

Reply:
good point, fixed.